linux_docker_resources/Dockerfile

```diff
@@ -12,7 +12,9 @@ ARG COMPILE_WITH_CLANG=false
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update && apt-get install --no-install-recommends -y locales
-RUN locale-gen en_US.UTF-8
+RUN if test ${UBUNTU_DISTRO} != buster; then locale-gen en_US.UTF-8; \
+  else sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+  dpkg-reconfigure --frontend=noninteractive locales; fi
```

Comment:
Please use two separate `RUN` lines here: one for Ubuntu, one for Debian.

Same below.

Reply:
Thank you for the review.
Changed as you pointed out.